Update Number of Tranches & Set Issuance Interval | Articles 21 & 22

Molebat opened the pull request:
Article 21: Change number of tranches from 12 to 10 to match whitepaper v1.2.

Article 22: Set a minimum issuance interval of 8-12 months. This is a guardrail for a more steady funding schedule, while allowing for some flexibility. Though it is unlikely that we would have multiple treasury funding proposals over a short timespan, this change would assuage the concerns of the community regarding "inflation shock".

Molebat committed:
Update Number of Tranches & Set Issuance Interval

Article 21: Change number of tranches from 12 to 10 to match whitepaper v1.2.

Article 22: Set a minimum issuance interval of 8-12 months. This is a guardrail for a more steady funding schedule, while allowing for some flexibility. Though it is unlikely that we would have multiple treasury funding proposals over a short timespan, this change would assuage the concerns of the community regarding "inflation shock".

hxrts commented:
Can we just make this a year instead of 8 months for simplicity? most of the ecosystem works on yearly timelines anyway.

Molebat committed:
Update: Set Issuance Interval | Article 22

Removed "There must be a minimum interval of 8 months between each successful proposal" and kept the calendar year limit. This was for simplicity. 

Used a calendar year limit instead of a 12-month interval. With a 12-month interval funding gets pushed back a bit every year unless its requested as soon as possible.

Molebat commented:
I've removed the 8 month part and kept the calendar year limit.
